spotify_project/tracks/views.py

```diff
@@ -0,0 +1,21 @@
+from django.shortcuts import get_object_or_404, render
+from tracks.models import Track
+
+
+def track_info(request, track_id):
+    track = get_object_or_404(Track, pk=track_id)
+    artists = list(track.artists.all())
+    seconds = (int(track.duration_ms) / 1000) % 60
+    minutes = (int(track.duration_ms) / (1000 * 60)) % 60
+    properties = {
```

Comment:
You can just pass in `track`, then in the template, just pull the things off of it you need. Only thing you need in the context is the track actually. You can make a property for duration and pull that too.